PowershellTools.Common/Constants.cs

```diff
@@ -16,8 +16,8 @@ internal static class Constants
 
         public const string ReadyEventPrefix = "VsPowershellToolProcess:";
 
-        // 10 seconds
-        public const int HostProcessStartupTimeout = 10000;
+        // 20 minutes
+        public const int HostProcessStartupTimeout = 20 * 60 * 1000; // wait for 20 minutes max for remote powershell host process to initialize

```

Comment:
True, I left it here in purpose as we are considering a retry logic as Huan mentioned.